### PR TITLE
TAG-22

### PR DESCRIPTION
**Pull request by KedarSki**

### CODE REVIEW

File: TagsterWebAPI/src/TagsterWebAPI/Controllers/AuthenticateController.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        public AuthenticateController(IJwtAuthenticationManager jwtAuthenticationManager)
+        {
+            _jwtAuthenticationManager = jwtAuthenticationManager;
+        }
+
+        public AuthenticateController(SignInManager<IdentityUser> signInManager)
+        {
+            _signInManager = signInManager;
+
+        }
```
Comment (flapek) [inline TagsterWebAPI/src/TagsterWebAPI/Controllers/AuthenticateController.cs:32]: make from this one constructor

**Review by flapek — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
-# tagster
\ No newline at end of file
+# Tagster
+
+Tagsster is a backend and frontend application that allows you to add tags to browsed profiles on [LinkedIn](https://www.linkedin.com).
```
Comment (kacperantoniak) [inline README.md:3]: "Tagster" instead of "Tagsster"

**Review by kacperantoniak — APPROVED** [review]

File: TagsterWebAPI/libraries/Tagster.Auth/Factory/TokenValidationFactory.cs [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                X509Certificate2 certificate = null;
+                var password = options.Certificate.Password;
+                var hasPassword = !string.IsNullOrWhiteSpace(password);
+                if (!string.IsNullOrWhiteSpace(options.Certificate.Location))
+                {
+                    certificate = hasPassword
+                        ? new X509Certificate2(options.Certificate.Location, password)
+                        : new X509Certificate2(options.Certificate.Location);
+                    var keyType = certificate.HasPrivateKey ? "with private key" : "with public key only";
+                    Console.WriteLine($"Loaded X.509 certificate from location: '{options.Certificate.Location}' {keyType}.");
```
Comment (orzeech) [inline TagsterWebAPI/libraries/Tagster.Auth/Factory/TokenValidationFactory.cs:57]: Here and in other places I'd use some kind of a configurable log provider. If yuo don't want to do it in the scope of this task, create a new one

File: TagsterWebAPI/libraries/Tagster.Auth/Services/IIdentityService.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Tagster.Auth.Dtos;
+using Tagster.Auth.Models;
+
+namespace Tagster.Auth.Services
+{
+    public interface IIdentityService
+    {
+        //Task<UserDto> GetAsync(Guid id, CancellationToken cancellationToken);
```
Comment (orzeech) [inline TagsterWebAPI/libraries/Tagster.Auth/Services/IIdentityService.cs:10]: do not commit commented-out code

File: TagsterWebAPI/libraries/Tagster.DataAccess/Migrations/20210905081434_Initial.Designer.cs [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+using Tagster.DataAccess.DBContexts;
+
+namespace Tagster.DataAccess.Migrations
+{
+    [DbContext(typeof(TagsterDbContext))]
+<<<<<<< HEAD:TagsterWebAPI/libraries/Tagster.DataAccess/Migrations/20210905081434_Initial.Designer.cs
+    [Migration("20210905081434_Initial")]
+=======
+    [Migration("20210906183605_Initial")]
+>>>>>>> e56f5e008b44f871bf3346eb31d0999174b491f7:TagsterWebAPI/libraries/Tagster.DataAccess/Migrations/20210906183605_Initial.Designer.cs
```
Comment (orzeech) [inline TagsterWebAPI/libraries/Tagster.DataAccess/Migrations/20210905081434_Initial.Designer.cs:17]: isn't something wrong with these?

**Review by orzeech — APPROVED** [review]

**Review by flapek — APPROVED** [review]

**Pull request merged**